src/Core/View.js

```diff
@@ -291,6 +310,29 @@ class View extends THREE.EventDispatcher {
         this.scene.traverse(ObjectRemovalHelper.cleanup);
     }
 
+
+
+    /**
+     * Update layer opacity depending on camera position.
+     *
+     * @param {Event} event Event used to get camera target position.
+     */
+    #updateTiledLayerOpacity(event) {
```

Comment:
This method would be better in `TiledGeometryLayer`

Reply:
Won't it be unusual to call this method from an eventlistener which is defined in `View` ?
I implemented it in `TiledGeometryLayer` but I don't find it optimized like that. 
What do you think ?